### ISSUE TITLE
Why did I redact this flag, breaking changes?

### ISSUE THREAD

**Issue by bjd2385** [bjd2385/gitlabci-lint-pre-commit-hook#19]
```text
Linting GitLab CI configs................................................Failed
- hook id: gitlabci-lint
- exit code: 2

usage: gitlabci-lint
       [-h]
       [-c CONFIGS]
       [-C GITLABCI_LINT_CONFIG]
       [-B [BASE_URL]]
       [--version]
       [-q]
gitlabci-lint: error: unrecognized arguments: -b https://git.ops.sbe-vision.com
```
